scenarios/csharp/dotnetcore/call-center/call-center/Program.cs

```diff
@@ -0,0 +1,408 @@
+            var transcriptionId = await CreateTranscription(this.userConfig.inputAudioURL);
+            await WaitForTranscription(transcriptionId);
+            var transcriptionUrl = await GetTranscriptionUri(transcriptionId);
+            var phrasesAndSpeakers = await GetTranscriptionPhrasesAndSpeakers(transcriptionUrl);
+            var phrases = phrasesAndSpeakers.Select(phraseAndSpeaker => phraseAndSpeaker.Item1).ToArray();
+            var sentiments = GetSentiments(phrases);
+            var conversationItems = TranscriptionPhrasesToConversationItems(phrasesAndSpeakers);
+            // NOTE: Conversation summary is currently in gated public preview. You can sign up here:
+            // https://aka.ms/applyforconversationsummarization/
+            var conversationAnalysisUrl = await RequestConversationAnalysis(conversationItems);
+            await WaitForConversationAnalysis(conversationAnalysisUrl);
+            var (conversationSummary, conversationPIIAnalysis) = await GetConversationAnalysis(conversationAnalysisUrl);
+            PrintResults(phrasesAndSpeakers, sentiments, conversationSummary, conversationPIIAnalysis);
+            // Clean up resources.
+            Console.WriteLine("Deleting transcription.");
+            await DeleteTranscription(transcriptionId);
+            return;
+        }
+
+        public static void Main(string[] args)
+        {
+            string usage = @"USAGE: dotnet run -- [...]
+
+  HELP
+    --help                          Show this help and stop.
+
+  CONNECTION
+    --speechKey KEY                 Your Azure Speech service subscription key. Required.
+    --speechRegion REGION           Your Azure Speech service region. Required.
+                                    Examples: westus, eastus
+    --languageKey KEY               Your Azure Cognitive Language subscription key. Required.
+    --languageEndpoint ENDPOINT     Your Azure Cognitive Language endpoint. Required.
+
+  LANGUAGE
+    --language LANGUAGE             The language to use for sentiment analysis and conversation analysis.
+                                    This should be a two-letter ISO 639-1 code.
+                                    Default: en
+    --locale LOCALE                 The locale to use for batch transcription of audio.
+                                    Default: en-US
+
+  INPUT
+    --input URL                     Input audio from URL. Required.
+    --stereo FORMAT                 Use stereo audio format.
+                                    If this is not present, mono is assumed.
+";
+            
+            try
+            {
+                if (args.Contains("--help"))
+                {
+                    Console.WriteLine(usage);
+                }
+                else
+                {
+                    Program program = new Program(args, usage);
+                    program.Run().Wait();
```

Comment:
>Wait [](http://example.com/codeflow?start=34&length=4)

I believe C# 8.0 (default for .NET Core 3.1) should allow you to make `Main` an `async Task Main` instead of `void Main` -- and use `await` directly. 

That might also make it convenient to just treat `Run` as a static method (to avoid mixing/matching static and instance concepts; an instance "Program" constructor is atypical).

Reply:
Thank you, I converted Main to public async static Task and got rid of the Program constructor. However, because I have an instance of UserConfig as a member of the Program class, I get the warning:

C:\Users\v-jasonwells\Documents\backup\code\call_center_20220419\cs\call-center\Program.cs(31,28): warning CS8618: Non-nullable field 'userConfig' must contain a non-null value when exiting constructor. Consider declaring the field as nullable. [C:\Users\v-jasonwells\Documents\backup\code\call_center_20220419\cs\call-center\call-center.csproj]

That happens even though I initialize it while new-ing Program:
```
new Program { userConfig = new UserConfig(args, usage) }
```

I'd rather not make userConfig nullable because then I have to null-check it every time I use it, unless that's the standard practice.